### DISCUSSION TITLE
TheHive to Cases migration in 2.3.100

### DISCUSSION THREAD

**Discussion by kiaraho** [Security-Onion-Solutions/securityonion#7089] [Unsupported Versions] [answered]
Which the inclusion of `Cases` in the new `2.3.100` release, what happens to existing `TheHive` investigations during the upgrade from `2.3.91`? Is there a way to migrate them?

**Comment by TOoSmOotH** [comment, marked as answer]
The Hive is still installed and you can access it from the left menu.

**Reply by SecurityNomad** [reply]
That is for the time being. However, would there be a means to migrate existing cases directly to the `Cases` tab without manually copying and pasting text?

**Reply by TOoSmOotH** [reply]
No but you can export the indices from The Hive's ES instance into the main SO instance and use Kibana to create dashboards to look at the old data but there are no plans to convert the cases over. I am sure if someone was inclined they could write something that did a simple migration of some of the key data.
